test/unit/math/binaryOps/src/binaryOps.cpp

```diff
@@ -0,0 +1,627 @@
+/** Copyright 2019 Axel Huebl, Benjamin Worpitz
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <alpaka/alpaka.hpp>
+#include <alpaka/test/MeasureKernelRunTime.hpp>
+#include <alpaka/test/acc/TestAccs.hpp>
+#include <alpaka/test/queue/Queue.hpp>
+
+#include <catch2/catch.hpp>
+#include <cmath>
+#include <iostream>
+#include "../../dataGen.hpp"
+
+/** List of all operators
+ * operator  | in std | definition |  range
+ * atan2     | Y | R^2\{(0,0)}
+ * fmod      | Y | R^2\{(x,0)|x in R}
+ * max       | Y | R^2
+ * min       | Y | R^2
+ * remainder | Y | R^2\{(x,0)|x in R}
+ *
+ * sincos is testes separately,
```

Comment:
- ... is "tested" ... 
- if sincos has a unique signature {(arg, &result1, &result2)}, then it is probably not needed to add another file like ternaryOps.cpp if there is only one candidate.